scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1525,7 +1530,19 @@ def addDBVersion(version):
         print("Exception occurred while adding db version : ", e)
         logging.error("Exception occurred while adding db version  : ", exc_info=True)
         raise e
-
+def verifyDBVersion(version):
+    try:
+        # fetch db_version from platform db
+        cur_platform.execute("select version_no from db_version order by updated_at desc limit 1")
+        db_version = cur_platform.fetchone()
+        if db_version is not None and db_version[0] != version:
+           raise Exception("Post upgrade could not proceed, upgrade version is not same existing schema")
+        else:
+           logging.info("Failed fetch the ISD Schema version from Platform DB")       
+    except Exception as e:
+        print("Exception occurred while adding db version : ", e)
```

Comment:
it should be exception occurred while fetching db version

Reply:
updated the Exception messages